docuploader/__main__.py

```diff
@@ -75,9 +75,12 @@ def upload(
         return sys.exit(1)
 
     if metadata_file is None:
-        metadata_path = pathlib.Path(documentation_path) / "docs.metadata"
-    else:
-        metadata_path = pathlib.Path(metadata_file)
+        metadata_file = "docs.metadata"
+        json_file = pathlib.Path(documentation_path) / "docs.metadata.json"
+        if json_file.exists():
+            docuploader.log.info("found docs.metadata.json instead")
```

Comment:
Either delete or delete "instead" (because there is no thing to be "instead of.")

Reply:
I'll delete it since this comment does not show up if the user passes in a `.json` file which is not consistent with this logging behavior